RNAdMobBanner.js

```diff
@@ -1,65 +1,54 @@
-import React, { Component } from 'react';
+import React, { useEffect, useState } from "react";
 import {
   requireNativeComponent,
   UIManager,
   findNodeHandle,
   ViewPropTypes,
-} from 'react-native';
-import { string, func, arrayOf } from 'prop-types';
+} from "react-native";
+import { string, func, arrayOf } from "prop-types";
 
-import { createErrorFromErrorData } from './utils';
+import { createErrorFromErrorData } from "./utils";
+function AdMobBanner(props) {
+  const [style, setStyle] = useState({});
 
-class AdMobBanner extends Component {
```

Comment:
Do we need to convert it to a function component? It will be harder to update from the mainstream

Reply:
I re-applied the changes from the original library, which included some fixes that you had made to the functional component itself. We can discuss this tomorrow - I don't mind leaving it the way it is.